screen.go

```diff
@@ -24,7 +24,9 @@ func ClearScreen() Msg {
 type clearScreenMsg struct{}
 
 // EnterAltScreen is a special command that tells the Bubble Tea program to
-// enter the alternate screen buffer.
+// enter the alternate screen buffer (i.e. the full terminal window). The
+// altscreen will be automatically exited when the program quits. To manually
+// exit the altscreen while the program is running, use [ExitAltScreen].
 //
```

Comment:
@meowgorithm what about also removing these lines ? This can lead to confusion :)